### ISSUE TITLE
Installation as module crashes with tsc: not found

### ISSUE THREAD

**Issue by phihag** [PeculiarVentures/node-webcrypto-ossl#94]
When I try to install node-webcrypto-ssl with `npm i node-webcrypto-ssl`, it breaks with the message `tsc: not found`:

```npm install node-webcrypto-ossl     

> node-webcrypto-ossl@1.0.17 install /dev/shm/bup/node_modules/node-webcrypto-ossl
> node-gyp rebuild

make: Entering directory '/dev/shm/bup/node_modules/node-webcrypto-ossl/build'
  CXX(target) Release/obj.target/nodessl/src/main.o
  *snip*
  CXX(target) Release/obj.target/nodessl/src/node/async_core.o
  SOLINK_MODULE(target) Release/obj.target/nodessl.node
  COPY Release/nodessl.node
make: Leaving directory '/dev/shm/bup/node_modules/node-webcrypto-ossl/build'

> node-webcrypto-ossl@1.0.17 postinstall /dev/shm/bup/node_modules/node-webcrypto-ossl
> npm run build


> node-webcrypto-ossl@1.0.17 build /dev/shm/bup/node_modules/node-webcrypto-ossl
> npm run build:es5


> node-webcrypto-ossl@1.0.17 build:es5 /dev/shm/bup/node_modules/node-webcrypto-ossl
> tsc

sh: 1: tsc: not found
```

Running `npm install typescript` works around the problem, but I believe typescript should be declared as a `dependency`, not only a `devDependency`.

**Comment by microshine** [comment]
Fixed in v1.0.19

**Comment by phihag** [comment]
Thank you very much! I can confirm that this update fixes my problem.
